### ISSUE TITLE
Set training=self.training in dropout

### ISSUE THREAD

**Issue by rao-ma** [BogiHsu/Tacotron2-PyTorch#8]
Line 120 in model.py

Should set training=self.training, otherwise in the evaluation, dropout function is still working.

**Comment by BogiHsu** [comment]
It's set to True on purpose as described in [here](https://github.com/NVIDIA/tacotron2/issues/247).
